index.html

```diff
@@ -49,10 +49,86 @@ <h2>LET'S CONNECT</h2>
+                    <li><div class="spans"></div></li>
+                    <li>Backend Dev</li>
+                    <li><p class="spans"></p></li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            <div class="techs">
+              <ul>
+                <li>HTML</li>
+                <li> CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button>See Projects</button>
+          </div>
+         </section>
+        
+         <section class="container-b">
+          <div class="bg-b">
+                <h1 class="about-h1">About Myself</h1>
+                
+                <p class="about-p">
+                  I'm a Software developer! Can help you build a product, feature or Website Look through Some of my Work and
+                  experience! if you like What you see and have a project you need coded, don't hesitate to contact me.
+                </p>
+                
+                <p class="connect">LET'S CONNECT</p>
+                
+                <ul class="socials">
+                  <li>
+                    <a href="https://twitter.com/ejama_isaac">
+                      <img src="img/Vector (3).svg" alt="">
+                    </a>
+                  </li>
+                  <li>
+                    <a href="https://www.linkedin.com/in/ejama-isaac-bab607267/"><img src="img/Linkedin icon.svg" alt=""></a>
+                  </li>
+                  <li>
+                    <a href="#"><img src="img/Vector (2).svg" alt=""></a>
+                  </li>
+                  <li>
+                    <a href=""><img src="img/Vector.svg" alt=""></a>
+                  </li>
+                  <li>
+                    <a href="https://github.com/isaac-ej"><img src="img/Vector (1).svg" alt=""></a>
+                  </li>
+                </ul>
+                <button class="btn">Get my resume</button>
+
+                <div class="techs">
+                <h3 class="under-lines">Languages <i class="fa-solid fa-chevron-down"></i></h3>
+
+                  <li class="lis"><img src="https://upload.wikimedia.org/wikipedia/commons/3/3b/Javascript_Logo.png" alt="">Javascript</li>
+                  <li class="lis"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/61/HTML5_logo_and_wordmark.svg/1200px-HTML5_logo_and_wordmark.svg.png" alt="">HTML</li>
+                  <li class="lis"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d5/CSS3_logo_and_wordmark.svg/1200px-CSS3_logo_and_wordmark.svg.png" alt="">CSS</li>
 
- 
+                <h3 class="under-lines">Frameworks<i class="fa-solid fa-chevron-right"></i></h3>
+                <h3 class="under-lines" >Skills<i class="fa-solid fas fa-chevron-right"></i></h3>
+                </div>
+              </section>
```

Comment:
- [ ] Great work on your progress! 👍🏿 However, I noticed that this section appears to be incomplete and some elements are missing from your design. I kindly suggest that you adhere to the Figma design and ensure that all the missing elements are added to complete this section. It's important to maintain consistency with the original design to ensure a cohesive and accurate representation. Keep up the good work and make sure to address these missing elements!

- [ ] Your Design

![image](https://github.com/Isaac-ej/Portfolio_Website/assets/68823331/5f11bcbb-c183-4dee-a4f2-5fbef34979aa)

- [ ] Figma Design
![image](https://github.com/Isaac-ej/Portfolio_Website/assets/68823331/fd72ac8f-3413-4ed1-92c2-1761fccdfa9e)